### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_Inspect: Role is incorrect for "Not Trusted" Control which is present at right of the menubar.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3996]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page opens with menubar.
3. Start Navigating through Menubar. 
4. Open Inspect Tool.
5. Verify "Role" or "Control Type" for "Not Trusted" control.

**Actual:**
Control Type for the "Not Trusted" button is given as Text which is incorrect.

**Expected:**
Control Type for the "Not Trusted" control should be button.

**UserImpact:**
The user will not be able to know the significance of the control.

**Attachment:**
![a11y_azurenotebooks_jupyter_menubar_inspect](https://user-images.githubusercontent.com/39204909/45813365-09f4a100-bcf0-11e8-8488-c3999e8effa8.png)

**Comment by takluyver** [comment]
Closed by #4693
